contrib/babelfishpg_tds/src/include/tds_request.h

```diff
@@ -15,14 +15,33 @@
 #ifndef TDS_REQUEST_H
 #define TDS_REQUEST_H
 
+#define BBF_NAMEDATALEND (NAMEDATALEN + 2)
+#define MD5_HASH_LEN 32
+
 #include "postgres.h"
+#include "port.h"
 
 #include "lib/stringinfo.h"
 #include "miscadmin.h"
 
 #include "src/include/tds_int.h"
 #include "src/include/tds_typeio.h"
 #include "src/collation.h"
+#include "catalog/pg_proc.h"
+#include "utils/syscache.h"
+#include "utils/catcache.h"
+#include "funcapi.h"
+#include "utils/acl.h"
+#include "utils/fmgroids.h"
+#include "access/table.h"
+#include "access/genam.h"
+#include "utils/lsyscache.h"
+#include "access/xact.h"
+#include "utils/varlena.h"
+#include "catalog/pg_type.h"
+#include "parser/scansup.h"
+#include "parser/parser.h"
```

Comment:
Too many unnecessary changes in tds extension

Reply:
Removed. moved it to pltsql_utils